build.gradle

```diff
@@ -16,16 +16,16 @@ ext {
 
 ext {
     // libs
-    automatticAboutVersion = '1.2.0'
+    automatticAboutVersion = '1.3.0'
     automatticRestVersion = '1.0.8'
-    automatticStoriesVersion = '2.3.0'
-    automatticTracksVersion = '3.2.0'
+    automatticStoriesVersion = '2.4.0'
+    automatticTracksVersion = '3.3.0'
     gutenbergMobileVersion = 'v1.103.1'
-    wordPressAztecVersion = 'v1.7.0'
-    wordPressFluxCVersion = 'trunk-f8fa03d500abff98dbf766816fc0fdfa72dc00ec'
-    wordPressLoginVersion = '1.5.0'
+    wordPressAztecVersion = 'v1.8.0'
```

Comment:
Bumping the Aztec version might introduce regressions due to changes incorporated in `1.8.0`. It would be great if we smoke-test the Gutenberg and Classic editor to avoid unexpected issues.

Reply:
Good point!